apps/io-wallet-user-func/asyncapi.yaml

```diff
@@ -0,0 +1,48 @@
+asyncapi: 3.0.0
+info:
+  title: IO Wallet - User Async API
+  version: 0.1.0
+channels:
+  walletInstanceRevoked:
+    address: "on-wallet-instance-revoked"
+    messages:
+      walletInstanceRevokedMessage:
+        contentType: application/json
+        payload:
+          type: object
+          properties:
+            id:
+              type: string
+  walletInstanceCreated:
+    address: "on-wallet-instance-created"
+    messages:
+      walletInstanceMessage:
+        contentType: application/json
+        payload:
+          type: object
+          properties:
+            id:
+              type: string
+            userId:
+              type: string
+            hardwareKey:
+              type: object
+              properties:
+                kty:
+                  type: string
+                x:
+                  type: string
+                y:
+                  type: string
+                crv:
+                  type: string
```

Comment:
Sometimes hardwareKey may contain optional attributes such as KID etc. What do you think about encoding the entire key in base64?

Reply:
OK for now. Then we settle in another PR